packages/auth/amplify_auth_cognito/example/integration_test/user_attributes_test.dart

```diff
@@ -22,7 +22,11 @@ void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
   group('User Attributes', () {
-    for (final environmentName in ['main', 'user-pool-only']) {
+    for (final environmentName in [
+      'main',
```

Comment:
nit: This set of strings is repeated a lot so am wondering if could be a constant somewhere? That would reduce the possibility of someone doing an incomplete update in the future which might cause a miss on test coverage.

Reply:
+1

Reply:
Good idea 👍